app/views/layouts/sumofus.slim

```diff
@@ -2,18 +2,22 @@ doctype html
 html
   head
     title  = @page.title
-    = render partial: 'shared/optimizely_snippet'
```

Comment:
The problem with the Optimizely snippet is that it slows down page load speed, but it must load before the rest of the page loads, otherwise there's flicker on the page from rendering the experiment changes. It really should be in the head.